modules/log/colors.go

```diff
@@ -290,7 +290,7 @@ func ColorFprintf(w io.Writer, format string, args ...interface{}) (int, error)
 		}
 		return fmt.Fprintf(w, format, v...)
 	}
-	return fmt.Fprintf(w, format)
+	return fmt.Fprintf(w, "%s", format)
```

Comment:
It seems this change is unnecessary.

Reply:
Kind of, `fmt.Fprintf(w, format)` would not print anything IIRC since there is no format string. `fmt.Fprintf(w, "%s", format)` adds this format string. But since all this is doing is returning the error, I adjusted it to just `fmt.Fprint(w, format)`.

Reply:
I don't think `fmt.Fprintf(w, format)` will not print anything, see https://play.golang.org/p/sjVCTMbTarA